README.md

```diff
@@ -70,4 +70,4 @@ docker run -e DB_DSN=mongodb://host:port -e JWT_SECRET_KEY=*your_secret_jwt_key*
 4. OpenAPI specifications
 
 #### Docker Hub
-https://hub.docker.com/r/resourcemod/rmod-registry
\ No newline at end of file
+https://hub.docker.com/r/resourcemod/rmod-registry
```

Comment:
This hasn't been changed? 😂

Reply:
Yeah, why should be? Thanks for your commit tho :)

Reply:
I didn't change it, so good to merge 👍